eitprocessing/features/breath_detection.py

```diff
@@ -295,7 +295,7 @@ def _remove_breaths_around_invalid_data(
 
         return breaths
 
-    def find_breaths(self, data: np.ndarray) -> list[Breath]:
+    def find_breaths_1d(self, data: np.ndarray) -> list[Breath]:
```

Comment:
Please don't rename functions that are used by other parts of the code or existing projects. This breaks all code using this feature.

Breaking code functionality should be done sparingly. In this case, I don't think it is necessary. 

Instead, if it is necessary to distinguish between data with different dimensionality, build a check into this function that is backwards compatible, but ensures it also handles new cases.